src/fmeval/eval_algorithms/faithfulness.py

```diff
@@ -28,19 +28,37 @@
 
 FAITHFULNESS = EvalAlgorithm.FAITHFULNESS.value
 
+RAW_VERDICTS = "raw_verdicts"
+STATEMENTS = "statements"
+RAW_STATEMENTS = "raw_statements"
+LONG_FORM_PROMPT = "long_form_prompt"
+NLI_STATEMENTS_PROMT = "nli_statements_prompt"
+QUESTION = "question"
+ANSWER = "answer"
 
 LONG_FORM_ANSWER_PROMPT = """\
-Human: Given a question and answer, create one or more statements from each sentence in the given answer. For each statement, please start with "Statement:".
-question:$question
+Human: You are given a question and its answer. Your task is to rewrite the answer into one or more simple and coherent statements. Make sure that each statement is faithful to the answer and begins with "Statement:".
+Please refer to the following example for the intended output format.
+<example>
+question: Who was Albert Einstein and what is he best known for?
+answer: He was a German-born theoretical physicist, widely acknowledged to be one of the greatest and most influential physicists of all time. He was best known for developing the theory of relativity, he also made important contributions to the development of the theory of quantum mechanics.
+statements:
+Statement: Albert Einstein was a German-born theoretical physicist.
+Statement: Albert Einstein is recognized as one of the greatest and most influential physicists of all time.
+Statement: Albert Einstein was best known for developing the theory of relativity.
+Statement: Albert Einstein also made important contributions to the development of the theory of quantum mechanics.
+</example>
+
+Now solve the following task.
+question: $question
 answer: $answer
 statements:\n Assistant:"""  # noqa: E501
 
 
 NLI_STATEMENTS_MESSAGE = """
-Human: Prompt: Natural language inference
-Consider the given context and following statements, then determine whether they are supported by the information present in the context. Provide a brief explanation for each statement before arriving at the verdict (Yes/No). Provide a final verdict for each statement in order at the end in the given format. Do not deviate from the specified format.
-
-Context:\nJohn is a student at XYZ University. He is pursuing a degree in Computer Science. He is enrolled in several courses this semester, including Data Structures, Algorithms, and Database Management. John is a diligent student and spends a significant amount of time studying and completing assignments. He often stays late in the library to work on his projects.
+Human: Your task is to judge the faithfulness of a series of statements based on a given context. For each statement you must return verdict as "Yes" if the statement can be directly inferred based on the context or "No" if the statement can not be directly inferred based on the context. In the end, always provide your final verdict for each statement in order. You are given an example below to demonstrate the intended output format.
```

Comment:
Actually, this instruction does not mention explanation but the example shows explanations. Do we need to update the examples?

Reply:
It looks like the model could generate the explanations as well. Since we don't use them, it might increase the cost of these evaluations because the LLM hosting service usually charge based on input and output tokens.

Reply:
Per offline discussion, will revisit prompt after testing.